components/cam/src/physics/cam/shoc.F90

```diff
@@ -1812,10 +1812,13 @@ subroutine x_y_terms_diag_third_shoc_moment(&
   !intent-outs
   real(rtype), intent(out) :: x0, y0, x1, y1
 
-  real(rtype), parameter :: a0=(0.52_rtype*c_diag_3rd_mom**(-2))/(c_diag_3rd_mom-2._rtype)
-  real(rtype), parameter :: a1=0.87_rtype/(c_diag_3rd_mom**2)
-  real(rtype), parameter :: a2=0.5_rtype/c_diag_3rd_mom
-  real(rtype), parameter :: a3=0.6_rtype/(c_diag_3rd_mom*(c_diag_3rd_mom-2._rtype))
+! local variables
+  real(rtype) :: a0, a1, a2, a3
+
+  a0=(0.52_rtype*(c_diag_3rd_mom**(-2._rtype)))/(c_diag_3rd_mom-2._rtype)
```

Comment:
Just saw this. We want `x^(-k)` to be impl'ed as `1/x^k`, so in this case `(1/bfb_square(c_diag_3rd_mom))`. The general principle is we avoid `pow` and `**` in every case that makes sense.